features/product/managing_product_variants/adding_product_variant.feature

```diff
@@ -89,3 +89,15 @@ Feature: Adding a new product variant
         Then I should be notified that it has been successfully created
         And the "VODKA_WYBOROWA_PREMIUM" variant of the "Wyborowa Vodka" product should appear in the store
         And the variant with code "VODKA_WYBOROWA_PREMIUM" should be priced at $0.00 for channel "United States"
+
+    @ui
```

Comment:
```suggestion
    @api @ui
```

Reply:
yeah, this part is still wip as there's no api steps implemented for this feature file